packages/lesswrong/components/ultraFeed/FeedContentBody.tsx

```diff
@@ -256,18 +244,29 @@ const FeedContentBody = ({
       if (hideSuffix) {
         suffix = '...';
       } else if (willTruncate && !isExpanded) {
-        // Determine which action to take when content is truncated:
-        // 1. If total content exceeds maxWordCount → show word count
-        // 2. If total content fits within maxWordCount → show "(read more)" or "(read again)"
+        const needsNavigation = wordCount > maxWordCount && continueReadingUrl;
+        
+        // Split on isRead because read items have significantly different UX on mobile:
+        // - Mobile positioning: read items show a positioned button at bottom-right instead of inline text
+        // - Separate desktop/mobile markup: read items use read-suffix-desktop/mobile classes
         if (isRead) {
-          // The truncated text needs ellipsis, Desktop shows action inline (via suffix), Mobile hides the suffix and shows action as positioned button
+          // Read items have special mobile UX: desktop shows inline text, mobile shows positioned button
           actionText = (wordCount > maxWordCount) ? `(read again, ${wordsRemaining} words →)` : '(read again)';
-          // Add both desktop suffix (hidden on mobile) and mobile ellipsis (hidden on desktop)
-          suffix = `<span class="read-suffix-mobile">...</span><span class="read-more-suffix read-suffix-desktop">... ${actionText}</span>`;
+          
+          if (needsNavigation) {
+            // Navigate to full content - clicking "read again, X words" navigates
+            suffix = `<span class="read-suffix-mobile">...</span><span class="read-more-suffix read-suffix-desktop read-more-navigate">... ${actionText}</span>`;
+          } else {
+            // Expand in place - clicking "read again" expands
+            suffix = `<span class="read-suffix-mobile">...</span><span class="read-more-suffix read-suffix-desktop">... ${actionText}</span>`;
```

Comment:
If the only difference here is the presence of `read-more-navigate` in the first case, that should probably be inlined:
```
suffix = `<span class="read-suffix-mobile">...</span><span class="read-more-suffix read-suffix-desktop${needsNavigation ? ' read-more-navigate' : ''}">... ${actionText}</span>`;
```

Reply:
(or, even better, pull `needsNavigation ? ' read-more-navigate' : ''` out to its own variable)